docs/operator/monitoring.md

```diff
@@ -345,8 +345,7 @@ access. So the monitoring is not limitted to data transferring.
 
 4) Transmit messages from message queue to Elastisearch
 
-This could be done via Logstash. Please refer to
-[https://www.elastic.co/blog/integrating-jms-with-elasticsearch-service-using-logstash].
+This could be done via Logstash. Please refer to [https://www.elastic.co/blog/integrating-jms-with-elasticsearch-service-using-logstash](https://www.elastic.co/blog/integrating-jms-with-elasticsearch-service-using-logstash).
```

Comment:
The old link seems to still be operational: https://www.elastic.co/blog/integrating-jms-with-elasticsearch-service-using-logstash

Reply:
Just had a look at this doc page on the site https://rucio.github.io/documentation/operator/monitoring/ and it seems like there's a few badly formatted links:
<img width="744" alt="image" src="https://github.com/rucio/documentation/assets/35903974/eb5c2333-6af0-4aa6-a8bd-b5c558c30d5f">

It's odd that it doesn't work properly with just square brackets around the URL; according to [this reference](https://ctan.org/help/markdown/links?lang=en) it should work, but it might just be a different flavour of Markdown.

I'm not a fan of seeing the URL written twice like that, I think in this context we should either do:

```suggestion
This could be done via Logstash. Please refer to (https://www.elastic.co/blog/integrating-jms-with-elasticsearch-service-using-logstash).
```
which is the way we do it here: https://github.com/rucio/documentation/blob/ddae8d1700752e4ce9ef4937558babebe01c2053/docs/operator/monitoring.md?plain=1#L82

Or:

```suggestion
This could be done via Logstash. Please refer to [this page](https://www.elastic.co/blog/integrating-jms-with-elasticsearch-service-using-logstash).
```

I think the second version looks a bit better

Reply:
Yeah you're totally right, looks like there's a ton that were written in a rst-like format that haven't been caught by the actions. I'll change this PR a little bit and try to grab all of those.